src/NuGet.Clients/NuGet.Console/WpfConsole/WpfConsole.cs

```diff
@@ -646,6 +652,18 @@ private void HideProgress()
                 (uint)100);
         }
 
+        private async Task HideProgressAsync()
+        {
+            await NuGetUIThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
+            var vsStatusBar = await GetVsStatusBarAsync();
+            vsStatusBar.Progress(
+                ref _pdwCookieForStatusBar,
```

Comment:
Would be great to have named arguments

Reply:
Sure. This is "Part1", I can change it to named arguments with "Part2".